.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: node_js
 sudo: required
 node_js:
-  - "0.10"
-  - "4.4.3"
+  - 4
```

Comment:
Node 4 [is EOL from today](https://medium.com/the-node-js-collection/april-2018-release-updates-from-the-node-js-project-71687e1f7742)